packages/components/src/theme/grid.scss

```diff
@@ -327,17 +336,33 @@ $table-cell-selection-bg-color: #EDF3FF;
 
     .cellular-input {
       border: $table-cell-selected-border;
+      display: block;
+      height: 100%;
+      max-height: $table-cell-max-height;
+      max-width: 400px;
       outline: none;
-      padding: 6px 2px;
+      padding: 7px 1px 2px 2px;
```

Comment:
minor: not sure if this is the right line, but when looking at a text field and number field next to each other in the cell display mode, it looked like the number was slightly closer to its right border than the left aligned text. Might be from the 1px here instead of 2px?

Reply:
I believe this is addressed as I've realigned all the padding declarations.

Reply:
Nice! This looks much better and looks to have also resolved the FF minor vertical "jump".